ytypes/leaf.go

```diff
@@ -62,6 +62,12 @@ func validateLeaf(inSchema *yang.Entry, value interface{}) (errors []error) {
 			return util.AppendErr(errors, fmt.Errorf("bad leaf type: expect Int64 for enum type for schema %s, have type %v",
 				schema.Name, ykind))
 		}
+	case reflect.Bool:
+		if ykind != yang.Yempty {
```

Comment:
&& ykind != yang.Ybool ?
If that was indeed missing could you please add test case?

Reply:
`yang.Ybool` is expected to be a ptr to a bool, whereas the empty is not since it can't be unset - it's really just there or not.